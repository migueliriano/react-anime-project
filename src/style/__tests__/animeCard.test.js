import React from 'react';

import {
  CardContainer,
  Header,
  BodyContainer,
} from '../animeCard';


describe('animeCard style components', () => {
  it('Should render a CardContainer Component', () => {
    const tree = renderer.create(<CardContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a Header Component', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a BodyContainer Component', () => {
    const tree = renderer.create(<BodyContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
