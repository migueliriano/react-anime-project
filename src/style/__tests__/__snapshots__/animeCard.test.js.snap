// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`animeCard style components Should render a BodyContainer Component 1`] = `
.c0 {
  padding: 10px;
  background-color: #f3f3f3;
  border: 1px solid #d0cdcd;
  line-height: 30px;
  border-radius: 2px;
}

<div
  className="c0"
/>
`;

exports[`animeCard style components Should render a CardContainer Component 1`] = `
.c0 {
  width: 60vw;
  margin-bottom: 30px;
}

@media (max-width:768px) {
  .c0 {
    width: 100%;
  }
}

<div
  className="c0"
/>
`;

exports[`animeCard style components Should render a Header Component 1`] = `
.c0 {
  padding-bottom: 10px;
  font-size: 20px;
  font-weight: bold;
}

<div
  className="c0"
/>
`;
